packages/flutter/lib/src/widgets/form.dart

```diff
@@ -349,6 +358,9 @@ class FormFieldState<T> extends State<FormField<T>> {
   /// True if this field has any validation errors.
   bool get hasError => _errorText != null;
 
+  /// True if the current value is valid without changing the state.
```

Comment:
nit: 

True if the current value is valid.

Calling this will not set [errorText] or [hasError] and it will also not display the error on screen. Use [validate] for that.

Reply:
And the docs of validate should also be updated to link to this property.

Reply:
Done.